docs/model-monitoring/monitoring-models-grafana.ipynb

```diff
@@ -17,36 +17,43 @@
     "\n",
     "```{note}\n",
     "You need to train and deploy a model to see results in the dashboards.\n",
     "The dashboards immediately display data if you already have a model that is trained and running with production data.\n",
     "```\n",
     "\n",
     "## Model monitoring Overview dashboard\n",
-    "The Overview dashboard displays the model endpoint IDs of a specific project. Only deployed models with Model Monitoring enabled are displayed.\n",
+    "The Overview dashboard displays the models and endpoint IDs of the selected project. Only deployed models with Model Monitoring enabled are displayed.\n",
     "Endpoint IDs are URIs used to provide access to performance data and drift detection statistics of a deployed model.\n",
     "\n",
     "![overview](../_static/images/overview.png)\n",
     "\n",
     "The Overview pane provides details about the performance of all the deployed and monitored models within a project. You can change projects by choosing a new project from the\n",
-    "**Project** dropdown. The Overview dashboard displays the number of endpoints in the project, the average predictions per second (using a 5-minute rolling average),\n",
-    "the average latency (using a 1-hour rolling average), and the total error count in the project.\n",
-    "\n",
-    "Additional details include:\n",
-    "* **Endpoint ID** &mdash; the ID of the deployed model. Use this link to drill down to the model performance and details panes.\n",
-    "* **Function** &mdash; the MLRun function to access the model\n",
-    "* **Model** &mdash; user defined name for the model\n",
-    "* **Model Class** &mdash; the implementation class that is used by the endpoint\n",
-    "* **First Request** &mdash; first request for production data\n",
-    "* **Last Request** &mdash; most recent request for production data\n",
-    "* **Error Count** &mdash; includes prediction process errors such as operational issues (for example, a function in a failed state), as well as data processing errors (for example, invalid timestamps, request ids, type mismatches etc.)\n",
-    "* **Drift Status** &mdash; no drift (green), possible drift (yellow), drift detected (red)\n",
-    "\n",
-    "At the bottom of the dashboard are heat maps for the Predictions per second (5 minute average), Average Latency (one hour average), and Errors.\n",
-    "See [How to Read a Heat Map](#how-to-read-a-heat-map) for more details.\n",
+    "**Project** dropdown. \n",
+    "\n",
+    "The top row of the Overview dashboard is a summary of:\n",
+    "* The number of endpoints in the project\n",
+    "* The average predictions per second (using a 5-minute rolling average),\n",
+    "* The average latency (using a 1-hour rolling average)\n",
+    "* The total error count in the project during the selected time period. \n",
+    "\n",
+    "The central table presents details about the endpoints in the project:\n",
```

Comment:
I would also add a comment at the end about the fact that this table is not reflected by the selected time range and just display the model endpoints status in the last 24h